charts/images.yaml

```diff
@@ -21,3 +22,36 @@ images:
   sourceRepository: github.com/gardener/machine-controller-manager
   repository: eu.gcr.io/gardener-project/gardener/machine-controller-manager
   tag: "v0.27.0"
+
+- name: csi-driver-disk
+  sourceRepository: github.com/kubernetes-sigs/azuredisk-csi-driver
+  repository: mcr.microsoft.com/k8s/csi/azuredisk-csi
+  tag: "v0.7.0"
+- name: csi-driver-file
```

Comment:
FYI: currently azurefile does not support restore from snapshot - ref https://github.com/kubernetes-sigs/azurefile-csi-driver/issues/136.

Reply:
(Already discussed via phone, just writing it down here so that others are aware as well): Not a blocker as snapshotting is a new feature that comes with CSI. So, everything that was working before still works after https://github.com/gardener/gardener-extension-provider-azure/pull/73#discussion_r427370421.